### DISCUSSION TITLE
Question about calculating the velocity of the moving interface in 2D CH model

### DISCUSSION THREAD

**Discussion by abc-hy** [idaholab/moose#20378] [Q&A Modules: Phase field] [answered]
Hi everyone,

I have some questions about calculating the velocity.
Is there any postprocessors that can be used to calculate the velocity of the the moving interface in a 2D CH model? Do I need to track the boundary's location at each time step and then calculate the velocity by myself? Or there is one commands that can directly calculate the velocity?

Thank you very much.
Best regards.

**Comment by laagesen** [comment, marked as answer]
Hi there- can you tell a little more about the geometry of your problem? If you know exactly where on the interface you want to calculate velocity, you could use the FindValueOnLine postprocessor to determine the position of the interface at each time step, then take the difference in position between two time steps and divide by dt (which you can get from csv output of all postprocessor data). It will be a little more tricky if you want to know the velocity of the interface at each position, so please let us know what geometry you want to consider.
